etc/cmake/BMQPlugins.cmake

```diff
@@ -0,0 +1,86 @@
+include_guard()
+
+macro(_bmq_dedup_link_flags target)
```

Comment:
We need to rethink how we're going to do what this dedup workaround does if we want to integrate it as the official way to add plugins into the upstream repo.